### ISSUE TITLE
add dynamic tags when sending the email

### ISSUE THREAD

**Issue by selul** [Codeinwp/mt-support-plugin#9]
@contactashish13 following this conv https://github.com/Codeinwp/themeisle/issues/449 i would like to add some dynamic tags to the users who are using the support form,

In gravity forms we have this notification -> http://prntscr.com/iq5hsj , i would like to add two mores there: 

1) {User lifetime value} - will get from wp customers table, the user lifetime value and return in the following format, **<50**,**(50,100)**,**(100,200)**, **(200+)**, **free** as strings if the lifetime value is between those limits. 
2) {Member since} - since when he is a customer, in this format 
 - **~1week** - if he is customer between 0 and 15 days. 
 - **~1month**  - if he is customer between 15 days and 60 
 - **~6months** - if he is customer between 150 days and 300
 - **~1year**  - if he is customer for 300 + 500 days
 - **++years**  - if he is customer for 500 + 

I think we can use this filter ( not sure )  -> https://docs.gravityforms.com/gform_merge_tag_filter/

**Comment by contactashish13** [comment]
@selul I did not understand the conversation https://github.com/Codeinwp/themeisle/issues/449. Do you want to add more notifications or more placeholders? Can you please specify the intent of this change?

**Comment by selul** [comment]
more placeholders. The intent is when the notification is sent ( i.e the email ) in the subject to have those specific pieces of pieces of information.

**Comment by contactashish13** [comment]
@selul can I assume that a customer is someone who has logged in before trying to use the support form?

**Comment by selul** [comment]
we assume that the user has an account, if he does not, we can return `not_customer` for both cases.
Regarding the lifetime value, in the edd_customers table you have a column for lifetime value, is the amount of $ that the customer spent on our products.

**Comment by contactashish13** [comment]
@selul should 60-150 days = ~3 months?

**Comment by selul** [comment]
yep.

**Comment by contactashish13** [comment]
@selul PR: https://github.com/Codeinwp/mt-support-plugin/pull/13

**Comment by selul** [comment]
@contactashish13 can you please add as a fallback if the user is not logged in, to get the email from the email field in the support form ?

**Comment by contactashish13** [comment]
@selul do you mean use that email field to determine the customer?

**Comment by selul** [comment]
exactly.

**Comment by contactashish13** [comment]
this form seems to contain the admin_email (recipient of the email, not the sender) and the email is not sent if this email Id is changed. 

![image](https://user-images.githubusercontent.com/12953439/37461239-ad4abb94-2873-11e8-9457-3ec053cec911.png)

**Comment by selul** [comment]
@contactashish13 that's because it's pulling the current user details, you need to be logged out to check that.

**Comment by contactashish13** [comment]
@selul no, it shows the same details when I'm logged out as well. And if Ii try to change the email ID in the form while submitting, the email disappears into a black hole.

**Comment by selul** [comment]
it might be something from the local environment, can we assume that the email field is filled in correctly and get the tags according to the field ( if is not logged in )

**Comment by contactashish13** [comment]
@selul done.
